packages/core/src/config/config-resolvers.ts

```diff
@@ -308,10 +322,11 @@ export async function resolvePlugins(
           !pluginModule.decorators.oas2 &&
           !pluginModule.decorators.async2 &&
           !pluginModule.decorators.async3 &&
-          !pluginModule.decorators.arazzo1
+          !pluginModule.decorators.arazzo1 &&
+          !pluginModule.decorators.overlay1
```

Comment:
I'm not sure there's a particular need in the overlay decorators or preprocessors, but let it be here for consistency.